### ISSUE TITLE
VM RAM usage keeps high after cloudlet finishes

### ISSUE THREAD

**Issue by manoelcampos** [cloudsimplus/cloudsimplus#429]
From @ulfaric:

The RAM issue always happens whenever you use a random function to generate RAM usage. The RAM usage of that VM will simply keeps as the last randomly generated value and doesn't change back to 0. If this is the last cloudlet to be scheduled on the VM.

My fix is simply add a never ending Cloudlet(using the on finish listener to put a new same Cloudlet back on to the VM) for each VM. Because I am simulating containers which must have a process that is always running. So this "deamon" cloudlet fixes the issue because it overwrites the incorrect RAM usage at the end.

**Comment by AndrewLawrence80** [comment]
Yes, and another bug is the VM RAM usage is 100% after a new VM is created

**Comment by jlxcsong** [comment]
您好！您发的邮件我已收到！

**Comment by manoelcampos** [comment]
That affects the example on #392
